tests/integration/sandbox.py

```diff
@@ -107,20 +107,23 @@ def raftlogidx(self):
     def dbfilename(self):
         return os.path.join(self.serverdir, self._dbfilename)
 
-    def cluster(self, *args):
+    def cluster(self, *args, expect_error=None):
         retries = self.up_timeout
         if retries is not None:
             retries *= 10
         while True:
             try:
                 return self.client.execute_command('RAFT.CLUSTER', *args)
             except redis.exceptions.RedisError as err:
-                LOG.info(err)
+                if expect_error is not None and expect_error in str(err):
+                    LOG.info(f"found expected error {expect_error}")
```

Comment:
The test code style uses `"string".format()`.

Reply:
"Standard is better than better". I don't mind this or the other but keep things consistent where possible.

Reply:
fixed all f"" strings